### PR TITLE
Removed phone number

### PR DESCRIPTION
**Pull request by Schnutek**
I changed the phone numbers as well as corrected text + line spacing issues on our Legal Notice Pages (DE, EN & NL). 

All four (for some reason there are two English versions) pages are now up to date.

### CODE REVIEW

File: src/pages/legal_notice.en.md [hunk 1]
```diff
@@ -12,11 +12,10 @@ Schlesische Straße 26<br />
 ###### Contact
 
 Phone: +49 (0)172 8463975<br />
-Phone: +49 (0)176 471 708 19<br />
 E-Mail: hello@honeypot.io<br />
 Managing Directors: Kaya Taner, Emma Tracey<br />
 Commercial Register: Amtsgericht Charlottenburg, HRB 167934 B<br />
-VAT-id: DE300856850
+VAT-ID: DE300856850
```
Comment (kdamball) [inline src/pages/legal_notice.en.md]: This info is missing on Dutch version

File: src/pages/legal_notice.de.md [hunk 2]
```diff
@@ -11,14 +11,11 @@ Schlesische Straße 26<br />
...
-
 Geschäftsführer: Kaya Taner, Emma Tracey<br />
 Handelsregister: Amtsgericht Charlottenburg, HRB 167934 B<br />
 USt-ID: DE300856850
 
 Verantwortlich für den Inhalt nach §55 Abs. 2 RStV:
-Kaya Taner<br />
-Schlesische Straße 26<br />
-10997 Berlin
+Kaya Taner, Emma Tracey; Adresse (siehe oben)<br />
```
Comment (kdamball) [inline src/pages/legal_notice.de.md]: If I follow the english version: shouldn't this be `(Adresse siehe oben)`

File: src/pages/legal_notice.en.md (hunk 1 above)
Comment (Schnutek) [reply src/pages/legal_notice.en.md]: @kdamball in dutch, VAT-ID, is called BTW-nummer. It's correct :)

File: src/pages/legal_notice.de.md (hunk 2 above)
Comment (Schnutek) [reply src/pages/legal_notice.de.md]: @kdamball talked to Tabea about it and we agreed that the way written there, makes the most sense in German.

**Pull request merged**